monitoring/monitoring.go

```diff
@@ -185,30 +187,30 @@ func getCouchbaseClouds(client *couchbasecloud.CouchbaseCloudClient, clouds map[
 			cloud := NewCouchbaseCloud()
 			cloud.ID = cloudResponse.Id
 			cloud.Name = cloudResponse.Name
-			cloud.Region = cloudResponse.Region
-			cloud.Provider = cloudResponse.Provider
-			cloud.Status = cloudResponse.Status
+			cloud.CloudRegion = NewCloudRegion(cloudResponse.Region)
+			cloud.Provider = string(cloudResponse.Provider)
+			cloud.Status = string(cloudResponse.Status)
 			cloud.VirtualNetworkCIDR = cloudResponse.VirtualNetworkCIDR
 			cloud.VirtualNetworkID = cloudResponse.VirtualNetworkID
 			clouds[cloud.ID] = cloud
 			cloudCount++
 		}
 
-		lastPage = listCloudsResponse.Cursor.Pages.Last
+		lastPage = *listCloudsResponse.Cursor.Pages.Last
```

Comment:
It's possible for `listCloudsResponse.Cursor.Pages.Last` to be nil which would trigger a nil pointer dereference, I pushed some changes to fix a couple of these comments